content/Arduino Cloud/General Cloud/Import-an-Arduino-Cloud-Template.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Import an Arduino Cloud Template"
+---
+
+In this article, you'll learn the step-by-step process for importing an Arduino Cloud template.
+
+## Import a Template
+
+1. Go to [IoT Templates](https://app.arduino.cc/templates) and select a template.
+
+1. Click the **Import Template** button.
+
+1. Plug in your device and wait for it to be detected. Once the device is detected, click on **Configure** to continue.
+
+    ![Dialog box displaying that the device was found](img/Device_found.png)
+
+1. Now that your device is detected and connected, the firmware of your Arduino device will be checked and updated if necessary to enhance connectivity. Follow the on-screen steps to proceed.
+
+    ![Dialog box displaying firmware updates](img/Checking_device_connectivity.png)
+
+1. Once the device setup is complete, enter your network credentials.
+
+    ![Dialog box for entering network credentials](img/Set_network_parameters.png)
+
+---
+
+## Still need help?
```

Comment:
The "Still need help?" heading should be used to provide additional solutions in articles that are already framed as troubleshooting. Example: ['Error: avrdude' when uploading](https://github.com/arduino/help-center-content/blob/98d433d448b693a5128f0dfd900d0013ebc1b769/content/Software%20Support/Upload/Error-avrdude-when-uploading.md?plain=1#L220-L231)

For "how to" articles, such as this one, use either "Troubleshooting" (general purpose) or a troubleshooting style title "If.. [issue]" (more specific).

```suggestion
## Troubleshooting
```